### PR TITLE
Add Kogan Air Purifier to device list

### PR DESCRIPTION
**Pull request by madchuska**
Add support for Kogan Air Purifier
All functions tested and working

### CODE REVIEW

File: custom_components/tuya_local/devices/kogan_air_purifier [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      mapping:
+        - dps_val: 1
+          value: "Sleep"
+        - dps_val: 2
+          value: "High"
+        - dps_val: 3
+          value: "Auto"
+secondary_entities:
+  - entity: sensor
+    name: Filter Remaining
```
Comment (make-all) [inline custom_components/tuya_local/devices/kogan_air_purifier:24]: HA convention is that only the first word in a name should be capitalised. (Same for Filter reset, Timer remaining, and Air quality below)

**Review by madchuska — COMMENTED** [review]
Updated

File: custom_components/tuya_local/devices/kogan_air_purifier [hunk 2]
```diff
@@ -40,11 +40,11 @@ secondary_entities:
           - dps_val: cancle
             value: "OFF"
           - dps_val: 1
-            value: "Low"
+            value: "low"
```
[the remark this answers is not in the document]
Comment (madchuska) [reply custom_components/tuya_local/devices/kogan_air_purifier]: Thanks, Done

File: custom_components/tuya_local/devices/kogan_air_purifier [hunk 3]
```diff
@@ -85,10 +85,10 @@ secondary_entities:
         type: string
         mapping:
           - dps_val: 1
-            value: "Good"
+            value: "good"
```
[the remark this answers is not in the document]
Comment (madchuska) [reply custom_components/tuya_local/devices/kogan_air_purifier]: have reverted to caps for now

**Review by madchuska — COMMENTED** [review]
Updated using light entity and formatting for UI

**Comment by madchuska** [comment]
@make-all - not sure if this is the correct way to do this (my first pull request)
but seems I missed adding the .yaml extension.
I can open another pull request if need be.

**Comment by make-all** [comment]
Sorry, I didn't notice before it was merged. It should be fixed now.

**Pull request merged**